src/lib.rs

```diff
@@ -75,10 +75,14 @@ mod engine;
 mod error;
 mod fn_call;
 mod fn_func;
-mod fn_native;
+pub mod fn_native;
```

Comment:
Avoid publicly exposing entire sub-modules.  Better use `pub use` later on to selectively export.

Reply:
I did it this way, because (a) almost everything was `pub`, and (b) everything that was `pub` was needed for actual plugin writing.

It seems much easier for module authors to import `rhai::fn_native::*` than the number of things that would need to be imported, including several traits.

Reply:
I wouldn't mind adding this to the `plugin` module as public exports either, so that they could import `rhai::plugin::*` to get everything. But the general idea is the same.

Reply:
Hhmmm... if you think you'll eventually need all public stuff under `plugins` then it makes sense.

However, you'll need to be careful to `pub(crate)` the stuff you don't want to expose...